modules/op-resolver-git-override-scm.adoc

```diff
@@ -5,7 +5,7 @@
 :_mod-docs-content-type: REFERENCE
 = Specifying a remote pipeline or task by using the Git resolver with the authenticated SCM API overriding the Git resolver configuration
 
-When creating a pipeline run, you can specify a remote pipeline from a Git repository by using the authenticated SCM API. When creating a pipeline or a task run, you can specify a remote task from a Git repository. You can override the initial configuration settings in specific pipeline runs or tasks to customize the behavior according to different use cases.
+You can override the initial configuration settings in specific pipeline runs or tasks to customize the behavior according to different use cases. You can use this method if no authenticated provider was configured.
```

Comment:
Hello @mramendi I added a line here as we discussed (I noted the suggestion you made, but then made a rebase mistake in another PR and after I finished dealing with that, I lost the note and forgot the exact line, so I apologise. :sweat: ) Could you please let me know if my tired brain remembered the use case here correctly? Thank you!